packages/mdc-textfield/icon/_mixins.scss

```diff
@@ -22,9 +22,9 @@
 
 // Public mixins
 
-@mixin mdc-text-field-icon-color($color) {
+@mixin mdc-text-field-icon-color($color, $onlyTrailing: false) {
```

Comment:
Is there a reason why we're singling out trailing like this?

Would it make more sense to have there be 2 color parameters? i.e. `$color, $trailingColor: $color` where if you specify one it applies to both, but if you specify two they apply to leading/trailing?

(Could also do `$trailingColor: null` and still conditionalize the output like you're currently doing.)

Reply:
This was only used to style the trailing icon when in an invalid state, since that's the only time the 2 icons have different colors. I don't think it makes sense to support passing in 2 colors, since majority of the time the icons will always be the same color. There's only 1 time we need an exception (and to be honest, I'm not sure why leading icons don't turn red when the text field is invalid).